app/util/ApplicationMixin.js

```diff
@@ -61,7 +61,7 @@ Ext.define('CpsiMapview.util.ApplicationMixin', {
         if (Ext.Array.some(me.serviceUrls, urlTest) === true) {
             if (Ext.Array.some(me.excludedUrls, urlTest) === false) {
 
-                var responseType = response.responseType ? response.responseType: response.getResponseHeader('content-type');
+                var responseType = response.responseType ? response.responseType: response.getResponseHeader ? response.getResponseHeader('content-type') : null;

```

Comment:
In what situations can null be returned here?

Reply:
mmm I can't recreate it now but I saw it happen, I thought it was when downloading an attachment

would you like me to roll this back and wait until I can recreate it?

Reply:
It would be good to check for null and Ext.log the response type to see when this comes up. Looks like it could be the kind of issue that would be hard to debug otherwise.